<?xml version="1.0" encoding="UTF-8"?><testrun name="PerfTests.for_2053_returns_false" project="Numbers" tests="2" started="2" failures="1" errors="0" ignored="0">
  <testsuite name="test.numbers.InfiniteNaturalNumberTest.DivisorsFinder.PrimeDivisors.HasOnlyOneDigitPrimeDivisors.TestResult.PerfTests" time="3.072">
    <testcase name="for_2048_returns_true" classname="test.numbers.InfiniteNaturalNumberTest.DivisorsFinder.PrimeDivisors.HasOnlyOneDigitPrimeDivisors.TestResult.PerfTests" time="0.0"/>
    <testcase name="for_2053_returns_false" classname="test.numbers.InfiniteNaturalNumberTest.DivisorsFinder.PrimeDivisors.HasOnlyOneDigitPrimeDivisors.TestResult.PerfTests" time="3.072">
      <failure>java.lang.AssertionError: 
Expected: is &lt;false&gt;
     but: was &lt;true&gt;&#13;
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)&#13;
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)&#13;
	at test.numbers.InfiniteNaturalNumberTest$DivisorsFinder$PrimeDivisors$HasOnlyOneDigitPrimeDivisors.computeAndCheck(InfiniteNaturalNumberTest.java:433)&#13;
	at test.numbers.InfiniteNaturalNumberTest$DivisorsFinder$PrimeDivisors$HasOnlyOneDigitPrimeDivisors$TestResult$PerfTests.after(InfiniteNaturalNumberTest.java:471)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:33)&#13;
	at de.bechte.junit.runners.context.statements.StatementExecutor.execute(StatementExecutor.java:28)&#13;
	at de.bechte.junit.runners.context.processing.MethodExecutor.run(MethodExecutor.java:83)&#13;
	at de.bechte.junit.runners.context.processing.MethodExecutor.run(MethodExecutor.java:57)&#13;
	at de.bechte.junit.runners.context.statements.RunChildren.evaluate(RunChildren.java:38)&#13;
	at de.bechte.junit.runners.context.statements.RunAll.evaluate(RunAll.java:27)&#13;
	at de.bechte.junit.runners.context.statements.StatementExecutor.execute(StatementExecutor.java:28)&#13;
	at de.bechte.junit.runners.context.HierarchicalContextRunner.run(HierarchicalContextRunner.java:134)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
  </testsuite>
</testrun>
